mindocr/models/backbones/rec_abinet_backbone.py

```diff
@@ -0,0 +1,73 @@
+import mindspore.nn as nn
+
+from ...models.utils.abinet_utils import Model, PositionAttention, ResTranformer
+from ._registry import register_backbone, register_backbone_class
+
+__all__ = [
+    "ABINetIterBackbone",
+    "abinet_backbone"]
+
+
+# ABINet_backbone
+
+
+@register_backbone_class
+class ABINetIterBackbone(nn.Cell):
```

Comment:
It looks that all `ABINetIterBackbone` does is call `BaseVision` class. Why don't just eliminate this middle man?

Reply:
Modified, thank you.